### PR TITLE
Disable automatic dark mode

### PR DESCRIPTION
**Pull request by jamilbk**
Disables automatic dark mode that follows the OS theme.

### CODE REVIEW

File: elixir/apps/web/assets/tailwind.config.js
Comment (jamilbk) [inline elixir/apps/web/assets/tailwind.config.js:1]: Hope you don't mind the prettier fixes :-). We'll probably need an a `.prettier-rc` or `.es-lint.rc` or something in the proj root.

Comment (AndrewDryga) [reply elixir/apps/web/assets/tailwind.config.js:1]: @jamilbk I also formatted the file but.. we will have a ton of conflicts. Maybe we should postpone this for now?

Comment (jamilbk) [reply elixir/apps/web/assets/tailwind.config.js:1]: K, I'll disable fix-on-save for the `elixir/` path

Comment (jamilbk) [reply elixir/apps/web/assets/tailwind.config.js:1]: Leaving this here for future reference: https://prettier.io/docs/en/configuration.html

**Review by jamilbk — COMMENTED** [review]
@andrewdryga Fixed.

**Comment by AndrewDryga** [comment]
@jamilbk I copy-pasted change to prevent rebases

**Pull request closed without merging**